Update hibernate-resume-windows.md

jain-ank opened the pull request:
Including details on how to enable hibernation on existing VMs.

jain-ank committed:
Update hibernate-resume-windows.md

Including details on how to enable hibernation on existing VMs.

mattmcinnes commented on articles/virtual-machines/windows/hibernate-resume-windows.md:
```diff
@@ -151,13 +148,89 @@ To learn more about REST, check out an [API example](/rest/api/compute/virtual-m
+> Ensure that the page file is located on the C: drive. If its not located on the C: drive, then first move it to the C: drive before proceeding with the steps below.
+
+
+### [CLI](#tab/enableWithCLI)
```
Replace with: 

`### [CLI](#tab/enableWithCLIExisting)`

jain-ank replied:
Fixed

mattmcinnes commented on articles/virtual-machines/windows/hibernate-resume-windows.md:
````diff
@@ -151,13 +148,89 @@ To learn more about REST, check out an [API example](/rest/api/compute/virtual-m
+      --name myVM \      
+   ```
+
+### [PowerShell](#tab/enableWithPS)
````
Replace with:

`### [PowerShell](#tab/enableWithPSExisting)`

jain-ank replied:
Fixed

mattmcinnes commented on articles/virtual-machines/windows/hibernate-resume-windows.md:
```diff
@@ -151,13 +148,89 @@ To learn more about REST, check out an [API example](/rest/api/compute/virtual-m
+
+## Enabling hibernation on an existing Windows VM 
+
+To enable hibernation on an existing VM, you can use Azure CLI, PowerShell and API. Before proceeding, ensure that the guest OS version supports hibernation on Azure. Refer to the list of [supported OS versions ](../hibernate-resume-windows.md#supported-windows-versions).
```
Remove ../ from link. Should present as:

`(hibernate-resume-windows.md#supported-windows-versions)`

jain-ank replied:
Fixed

jain-ank committed:
Update hibernate-resume-windows.md

Fixed the errors

jain-ank committed:
Update hibernate-resume-intro.md

Removing Preview terms as we get this doc ready for GA announcement during May 21 week
